uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -203,6 +203,37 @@ <h4 class="modal-title">Leave Request</h4>
                 </div>
               </div>
             </crm-loading>
+            <div class="row" ng-show="$ctrl.sickness == true">
+              <div class="col-xs-12 form-horizontal">
+              <hr>
+                <div class="form-group">
+                  <label class="col-xs-4 control-label">Reason<i class="fa fa-asterisk"></i></label>
+                  <div class="col-xs-8">
+                    <div class="chr_custom-select chr_custom-select--full">
+                      <select name="reason" class="ng-pristine ng-valid ng-not-empty ng-touched" id="">
+                        <option value="- select -">- select -</option>
+                      </select>
+                    </div>
+                  </div>
+                </div>
+                <div class="form-group">
+                  <div class="col-xs-12">
+                    <label class="chr_custom-checkbox squared">
```

Comment:
Remove any custom styles for checkbox and use default one, please remove styles from the theme PR also